Sources/Live/LiveUIApplier.swift

```diff
@@ -21,6 +21,8 @@ public class ReactantLiveUIViewApplier {
     private let resolveStyle: (UIElement) throws -> [Property]
     private let setConstraint: (String, SnapKit.Constraint) -> Bool
 
+    public typealias ViewTuple = (name: String, element: UIElement, view: UIView)
```

Comment:
I'd maybe rename it to something like `IdentifiedView` to make its purpose more clear.